core/variant/variant_utility.cpp

```diff
@@ -383,11 +383,11 @@ struct VariantUtilityFunctions {
 		return value;
 	}
 
-	static inline double clampf(double x, double min, double max) {
+	static inline double clampf(double x, double min = 0.0, double max = 1.0) {
```

Comment:
I dont think clamp should have default arguments

Reply:
Its like min and max having them. Its more error prone than useful.

Reply:
Ok, I updated the PR so that it doesn't do that.